### PR TITLE
Change text on HM modal 

### PR DESCRIPTION
**Pull request by janKozie1**
for: https://app.zenhub.com/workspaces/openstax-unified-5b71aabe3815ff014b102258/issues/openstax/unified/1103

### CODE REVIEW

File: src/app/content/highlights/components/HighlightStyles.tsx [hunk 1]
```diff
@@ -129,28 +123,8 @@ export const ImageWrapper = styled.div`
   width: ${(desktopPopupWidth - popupBodyPadding) / 2}rem;
 `;
 
-// tslint:disable-next-line: variable-name
-export const StickyNoteBullet = styled.div`
```
Comment (janKozie1) [inline src/app/content/highlights/components/HighlightStyles.tsx:133]: Is there a reason why it was left out?

Comment (KarinaMendez2) [reply src/app/content/highlights/components/HighlightStyles.tsx:133]: Why were the bullets removed? @TomWoodward

**Review by KarinaMendez2 — APPROVED** [review]

File: src/app/content/highlights/components/HighlightStyles.tsx (hunk 1 above)
Comment (TomWoodward) [reply src/app/content/highlights/components/HighlightStyles.tsx:133]: i think the "bullet" you removed is actually the point on the sticky note, which we want to keep

**Review by TomWoodward — CHANGES_REQUESTED** [review]

**Comment by janKozie1** [comment]
Should the styles for a tooltip be removed? They are not used anywhere after this PR gets merged.

**Comment by TomWoodward** [comment]
yes the tooltip styles can be removed

**Comment by TomWoodward** [comment]
also i had asked karina to re-sort the messages file so you are now getting conflicts, sorry about that

**Review by TomWoodward — APPROVED** [review]

**Pull request merged**